processing/src/main/java/org/apache/druid/query/QueryContext.java

```diff
@@ -577,6 +577,14 @@ public String getBrokerServiceName()
     return getString(QueryContexts.BROKER_SERVICE_NAME);
   }
 
+  public long getMaxInputBytesPerWorker(long defaultValue)
```

Comment:
This seems weird. IMO as this is a MSQ specific change, query context should not be touched. 
"ControllerImpl#initializeQueryDefAndState" should have the changes for parsing the value from the key value context map. 
Checkout `MultiStageQueryContext.isFaultToleranceEnabled(queryContext);` as an example.

Reply:
Moved to MSQQueryContext